frontend/src/pages/Login.jsx

```diff
@@ -1,9 +1,52 @@
-function Login() {
-	return ( 
-		<div className="bg-dark min-h-screen pt-36 text-white">
-			<h1>login</h1>
-		</div>
-	 );
+import React from "react";
+import Navbar from "../components/Navbar";
+
+const BUTTON_CONTAINER = "bg-primary w-96 m-auto text-xl px-15 py-3 rounded-lg hover:text-dark ease-in duration-300 font-bold"
+
+const Login = () => {
```

Comment:
Since the Login in has text fields, you need to use ``useState`` to keep track of what the user enters for the email and password field. Checkout the code I wrote for the Checkout and Reviews page to see how I did it